toplevel/topdirs.ml

```diff
@@ -143,9 +143,13 @@ let load_file = Topeval.load_file false
 
 (* Load commands from a file *)
 
-let dir_use ppf name = ignore(Toploop.use_file ppf name)
+let dir_use ppf name =
+  let usage = File name in
+  ignore (Toploop.use_file ppf usage)
```

Comment:
Here `usage` is weird; maybe you could directly use `ignore (Toploop.use_input ppf (File name))`?